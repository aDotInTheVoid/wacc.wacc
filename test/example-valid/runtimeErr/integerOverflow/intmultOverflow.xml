<unit>
  <main>
    <addr_local>0</addr_local>
    <num>2147483</num>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
    <addr_local>0</addr_local>
    <local>0</local>
    <num>1000</num>
    <op>mul</op>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
    <addr_local>0</addr_local>
    <local>0</local>
    <num>1000</num>
    <op>mul</op>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
    <addr_local>0</addr_local>
    <local>0</local>
    <num>1000</num>
    <op>mul</op>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
  </main>
</unit>