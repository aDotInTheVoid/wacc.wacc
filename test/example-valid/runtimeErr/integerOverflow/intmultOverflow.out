2147483
2147483000
-648000
-648000000
